bulletroboy/operator.py

```diff
@@ -22,8 +22,14 @@ def __init__(self, body_id):
 		self.links = self.get_links()
 		self.movements = Movements(self)
 		self.ef_publisher = self.create_publisher(PoseStamped, '/roboy/simulation/operator/pose/endeffector', 10)
+		self.srv = self.create_service(HumanLinkIdFromName, 'human_link_id_from_link_name', self.human_link_id_from_link_name)
 		
 
+	def human_link_id_from_link_name(self, request, response):
```

Comment:
it is in the operator class, so I believe there is no need to prepend human to the name